% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
]{book}
\usepackage{amsmath,amssymb}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math} % this also loads fontspec
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
\usepackage{lmodern}
\ifPDFTeX\else
  % xetex/luatex font selection
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\usepackage{longtable,booktabs,array}
\usepackage{calc} % for calculating minipage widths
% Correct order of tables after \paragraph or \subparagraph
\usepackage{etoolbox}
\makeatletter
\patchcmd\longtable{\par}{\if@noskipsec\mbox{}\fi\par}{}{}
\makeatother
% Allow footnotes in longtable head/foot
\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}
\makesavenoteenv{longtable}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{5}
\usepackage{booktabs}

\usepackage{color}
\usepackage{framed}
\setlength{\fboxsep}{.8em}

% These colours were manually entered, they shouldn't matter unless you want pdf output

\newenvironment{redbox}{
  \definecolor{shadecolor}{RGB}{243, 154, 157}
  \color{white}
  \begin{shaded}}
 {\end{shaded}}

\newenvironment{bluebox}{
  \definecolor{shadecolor}{RGB}{172, 210, 237}
  \color{white}
  \begin{shaded}}
 {\end{shaded}}

\newenvironment{greenbox}{
  \definecolor{shadecolor}{RGB}{141, 181, 128}
  \color{white}
  \begin{shaded}}
 {\end{shaded}}
\ifLuaTeX
  \usepackage{selnolig}  % disable illegal ligatures
\fi
\usepackage[]{natbib}
\bibliographystyle{plainnat}
\usepackage{bookmark}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\urlstyle{same}
\hypersetup{
  pdftitle={Metabolomics 2023},
  pdfauthor={Faculty: David Wishart, Jianguo Xia, Zhiqiang Pang, Jessica Ewald, Lun Zhang, Mark Berkjanskii, Michelle Brazas, and Nia Hughes},
  hidelinks,
  pdfcreator={LaTeX via pandoc}}

\title{Metabolomics 2023}
\author{Faculty: David Wishart, Jianguo Xia, Zhiqiang Pang, Jessica Ewald, Lun Zhang, Mark Berkjanskii, Michelle Brazas, and Nia Hughes}
\date{July 6, 2023 - July 7, 2023}

\begin{document}
\maketitle

{
\setcounter{tocdepth}{1}
\tableofcontents
}
\part{Introduction}\label{part-introduction}

\chapter{Workshop Info}\label{workshop-info}

Welcome to the 2023 Metabolomics Canadian Bioinformatics Workshop webpage!

\section{Schedule}\label{schedule}

\subsection{Edmonton}\label{edmonton}

\begin{longtable}[]{@{}
  >{\centering\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.1117}}
  >{\centering\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.3564}}
  >{\centering\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.1117}}
  >{\centering\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.4202}}@{}}
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\centering
\textbf{Time (Mountain)}
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
\textbf{July 6}
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
\textbf{Time (Mountain)}
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
\textbf{July 7}
\end{minipage} \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
8:00 & Arrivals \& Check-in & 8:00 & Arrivals \\
8:30 & Welcome (Nia Hughes) & 8:30 & \\
9:00 & Module 1: Introduction to Metabolomics (David Wishart) & 9:00 & Module 5: Backgrounder in Omics Data Science (Jeff Xia) \\
10:30 & Break (45 mins) & 10:30 & Break (45 mins) \\
11:15 & Module 2: Targeted, Quantitative Metabolomics (David Wishart) & 11:15 & Module 6: Data Analytics for Untargeted Metabolomics \\
12:15 & Lunch (45 mins) & 12:15 & Lunch (45 mins) \\
13:00 & Module 3 (Lab): Quantitative Metabolomics (David Wishart) & 13:00 & Module 7 (Lab): Metabolomics Data Analysis using MetaboAnalyst 5.0 (Jeff Xia) \\
15:00 & Break (30 mins) & 15:00 & Break (30 mins) \\
15:30 & Module 4: Databases for Biological Interpretation (David Wishart) & 15:30 & Module 8: Integrating Metabolomics with other Omics (Jeff Xia) \\
17:00 & Finished & 17:00 & Survey \& Closing Remarks \\
\end{longtable}

\subsection{Montréal}\label{montruxe9al}

\begin{longtable}[]{@{}
  >{\centering\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.1075}}
  >{\centering\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.3602}}
  >{\centering\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.1075}}
  >{\centering\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.4247}}@{}}
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\centering
\textbf{Time (Eastern)}
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
\textbf{July 6}
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
\textbf{Time (Eastern)}
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
\textbf{July 7}
\end{minipage} \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
10:00 & Arrivals \& Check-in & 10:00 & Arrivals \\
10:30 & Welcome (Nia Hughes) & 10:30 & \\
11:00 & Module 1: Introduction to Metabolomics (David Wishart) & 11:00 & Module 5: Backgrounder in Omics Data Science (Jeff Xia) \\
12:30 & Lunch (45 mins) & 12:30 & Lunch (45 mins) \\
13:15 & Module 2: Targeted, Quantitative Metabolomic (David Wishart) & 13:15 & Module 6: Data Analytics for Untargeted Metabolomics \\
14:15 & Break (45 mins) & 14:15 & Break (45 mins) \\
15:00 & Module 3 (Lab): Quantitative Metabolomics (David Wishart) & 15:00 & Module 7 (Lab): Metabolomics Data Analysis using MetaboAnalyst 5.0 (Jeff Xia) \\
17:00 & Break (30 mins) & 17:00 & Break (30 mins) \\
17:30 & Module 4: Databases for Biological Interpretation (David Wishart) & 17:30 & Module 8: Integrating Metabolomics with other Omics (Jeff Xia) \\
19:00 & Finished & 19:00 & Survey \& Closing Remarks \\
\end{longtable}

\section{Pre-work}\label{pre-work}

\href{https://docs.google.com/forms/d/e/1FAIpQLSezV8cbusyLYa98AJ50sZfOlW1HsKZmDh7XnsDKusRg73zkbw/viewform}{You can find your pre-work here.}

\chapter{Meet Your Faculty}\label{meet-your-faculty}

\subsubsection{David Wishart}\label{david-wishart}

\begin{quote}
Distinguished University Professor of Biological Science and Computing Science
University of Alberta
Edmonton, AB, CA

--- \href{mailto:dwishart@ualberta.ca}{\nolinkurl{dwishart@ualberta.ca}}
\url{www.wishartlab.com}
\end{quote}

David was one of the co-founders of the CBW in 1998. He has active research interests in the
application of bioinformatics and statistics to a wide range of computational biology problems,
from genomics to proteomics to metabolomics. In addition to running a large computational
biology lab, David also operates a large wet lab (analytical chemistry, molecular biology,
nanotechnology, cell biology, structural biology) supported by its own fabrication shop and
electrical engineering group.

\subsubsection{Jianguo (Jeff) Xia}\label{jianguo-jeff-xia}

\begin{quote}
Canada Research Chair in Bioinformatics and Big Data Analysis
Institute of Parasitology, McGill University
Montréal, QC, CA

--- \href{mailto:jeff.xia@cmgill.ca}{\nolinkurl{jeff.xia@cmgill.ca}}
\url{www.xialab.ca}
\end{quote}

Jeff has a broad background in medicine, immunology and bioinformatics. He has been involved
in CBW since 2011. Jeff has active research interests in metabolomics, transcriptomics,
microbiomics and multi-omics integration, with a key focus on democratizing omics data
analytics via web/cloud - based platforms and visualization.

\subsubsection{Zhiqiang Pang}\label{zhiqiang-pang}

\begin{quote}
PhD Candidate
Institute of Parasitology, McGill University
Montréal, QC, CA

--- \href{mailto:zhiqiang.pang@mail.mcgill.ca}{\nolinkurl{zhiqiang.pang@mail.mcgill.ca}}
\end{quote}

Zhiqiang is a PhD candidate in the Xia Lab at McGill University.
He graduated from Jilin University with his Bachelor's degree from
School of Pharmaceutical Sciences (2016) and Master's degree from the College of Basic
Medical Sciences (2019). His research project is about optimizing metabolomics raw data
processing to gain deep biological insight on infectious diseases, cancer and host-microbiome
interactions.

\subsubsection{Jessica Ewald}\label{jessica-ewald}

\begin{quote}
Postdoctoral Fellow
Institute of Parasitology, McGill University
Montréal, QC, CA

--- \href{mailto:jessica.ewald@mail.mcgill.ca}{\nolinkurl{jessica.ewald@mail.mcgill.ca}}
\end{quote}

Jessica is a postdoctoral fellow in the Xia Lab at McGill University
where she is currently studying multi-omics integration. Her
background includes an undergraduate degree in Environmental
Engineering and Applied Mathematics (Harvard College, 2017) and
a PhD in Bioinformatics (McGill University, 2022). Overall, her research has focused on using
computational approaches to understand the impact of environmental contaminants on human
and ecosystem health.

\subsubsection{Lun (Allen) Zhang}\label{lun-allen-zhang}

\begin{quote}
Research Associate
Department of the Biological Sciences, University of Alberta
Edmonton, AB, CA

--- \href{mailto:lun2@ualberta.ca}{\nolinkurl{lun2@ualberta.ca}}
\end{quote}

Dr.~Lun Zhang is a Research Associate in the Wishart lab at the
University of Alberta. Since joining the lab in 2017, he has mainly
focused on the development of LC-MS-based metabolomics
assays and software. He also participated in multiple translational
medicine research projects, such as discovering the diagnostic biomarkers for lung cancer,
colorectal cancer and polyps.

\subsubsection{Mark Berjanskii}\label{mark-berjanskii}

\begin{quote}
Research Associate
Department of the Biological Sciences, University of Alberta
Edmonton, AB, CA

--- \href{mailto:mb1@ualberta.ca}{\nolinkurl{mb1@ualberta.ca}}
\end{quote}

Mark obtained a Ph.D.~in Biochemistry at University of
Missouri-Columbia, USA. Mark's research interests include NMR
metabolomics, bioinformatics, protein NMR structure determination,
interactions, misfolding, and dynamics. Between 1996 and 2004, he
worked as a member of several research teams that studied the Hsp70-Hsp40 chaperone
system. Mark joined Dr.~Wishart's group at University of Alberta in 2004. Between 2005 and
2013, Mark studied prion proteins that, when misfolded, cause Mad Cow Disease in cattle and
similar Creutzfeldt-Jakob Disease in humans. Since 2004, he has been involved in developing
several programs for analysis of protein structure and dynamics, such as Random Coil Index,
Preditor, GeNMR, CS23D, PROSESS, Resolution-by-proxy, and Gamdy, as well as
bioinformatic analysis and NMR metabolomics.

\subsubsection{Michelle Brazas}\label{michelle-brazas}

\begin{quote}
Acting Scientific Director
Canadian Bioinformatics Workshop (CBW)
Toronto, ON, CA

--- \href{mailto:supporting@bioinformatics.ca}{\nolinkurl{supporting@bioinformatics.ca}}
\end{quote}

Dr.~Michelle Brazas is the Associate Director for Adaptive Oncology at
the Ontario Institute for Cancer Research (OICR), and acting Scientific
Director at Bioinformatics.ca. Previously, Dr.~Brazas was the Program Manager for
Bioinformatics.ca and a faculty member in Biotechnology at BCIT. Michelle co-founded and runs
the Toronto Bioinformatics User Group (TorBUG) now in its 11th season, and plays an active
role in the International Society of Computational Biology where she sits on the Board of
Directors and Executive Board.

\subsubsection{Nia Hughes}\label{nia-hughes}

\begin{quote}
Program Manager, Bioinformatics.ca
Ontario Institute for Cancer Research (CBW)
Toronto, ON, CA

--- \href{mailto:nia.hughes@oicr.on.ca}{\nolinkurl{nia.hughes@oicr.on.ca}}
\end{quote}

Nia is the Program Manager for Bioinformatics.ca, where she coordinates
the Canadian Bioinformatics Workshop Series. Prior to starting at OICR, she completed her
M.Sc. in Bioinformatics from the University of Guelph in 2020 before working there as a
bioinformatician studying epigenetic and transcriptomic patterns across maize varieties.

\chapter{Data}\label{data}

\subsubsection{Course Data Downloads}\label{course-data-downloads}

\begin{itemize}
\tightlist
\item
  \href{https://drive.google.com/file/d/1cvs1t7__xR6u4U7FNM2oht17Om6E3ZaF/view}{LC-MS lung cancer data}
\item
  \href{https://drive.google.com/file/d/1tGKu6pPbBWd3MAFtWSn5kxV0cvjX_mab/view}{GC-MS sleep apnea data}
\item
  \href{https://drive.google.com/file/d/1GvRMVJZkVdzikDZC4RDB-ef22_Px4Xs_/view}{NMR sheep pregnancy data}
\end{itemize}

\part{Modules}\label{part-modules}

\chapter{Module 1: Introduction to Metabolomics}\label{module-1-introduction-to-metabolomics}

\section{Lecture}\label{lecture}

\chapter{Module 2: Targeted, Quantitative Metabolomics}\label{module-2-targeted-quantitative-metabolomics}

\section{Lecture}\label{lecture-1}

\chapter{Module 3: Quantitative Metabolomics}\label{module-3-quantitative-metabolomics}

\section{Lab}\label{lab}

\chapter{Module 4: Databases for Biological Interpretation}\label{module-4-databases-for-biological-interpretation}

\section{Lecture}\label{lecture-2}

\chapter{Module 5: Backgroun in Omics Data Science}\label{module-5-backgroun-in-omics-data-science}

\section{Lecture}\label{lecture-3}

\chapter{Module 6: Data Analytics for Untargeted Metabolomics}\label{module-6-data-analytics-for-untargeted-metabolomics}

\section{Lecture}\label{lecture-4}

\chapter{Module 7: Metabolomics Data Analysis using MetaboAnalyst 5.0}\label{module-7-metabolomics-data-analysis-using-metaboanalyst-5.0}

\section{Lab}\label{lab-1}

\chapter{Module 8: Integrating Metabolomics with Other Omics}\label{module-8-integrating-metabolomics-with-other-omics}

\section{Lecture}\label{lecture-5}

  \bibliography{book.bib,packages.bib}

\end{document}
